modules/search/class-jetpack-instant-search.php

```diff
@@ -410,6 +410,10 @@ public function auto_config_search() {
 		$sidebar_id            = false;
 		$sidebar_searchbox_idx = false;
 		if ( $has_sidebar ) {
+			if ( empty( $sidebars['sidebar-1'] ) ) {
+				// Adding to an empty sidebar is generally a bad idea.
+				$has_sidebar = false;
```

Comment:
Can't see this variable being used anywhere else. Not even sure I understand the code, but shouldn't this return false or something similar?

Reply:
We use it here:
https://github.com/Automattic/jetpack/blob/master/modules/search/class-jetpack-instant-search.php#L456

Reply:
Ignore me. 😬